content/kubernetes/release-notes/6-4-2-releases/k8s-6-4-2-4.md

```diff
@@ -122,6 +122,10 @@ The following table shows supported distributions at the time of this release. Y
 
 ## Known limitations
 
+* REC creation fails in form view on OperatorHub. (RED-100517)
```

Comment:
Let's highlight that this is OpenShift only (not everyone knows what OperatorHub mean).
Maybe:

> REC creation via OpenShift's OperatorHub form view fails.